common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -159,6 +166,8 @@ def test_full_name_field(self):
             [u'another name', self.USERNAME],
         )
 
+        self.assert_event_emitted_num_times('edx.user.settings.changed', self.start_time, self.user_id, 2)
```

Comment:
Why 2 times?

I was imagining you would test the actual data in the event, but perhaps that is not necessary if you have good unit test coverage.